pkg/restapi/oidc/operations.go

```diff
@@ -165,6 +165,13 @@ func (o *Operation) oidcLoginHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	_, found := session.Get(userSubCookieName)
+	if found {
+		http.Redirect(w, r, o.walletDashboard, http.StatusMovedPermanently)
```

Comment:
@soluchok status code should not be MovedPermanently. Should use `http.StatusFound`